### PR TITLE
Add icon for ServiceStore

### PR DESCRIPTION
**Pull request by gayathrir11**
## How did you test this change?

- [ ] Test(s) added
- [ ] Manual testing (please provide screenshots/recordings)
- [x] No testing (please provide an explanation)
NA

### CODE REVIEW

File: packages/legend-extension-external-store-service/package.json [hunk 1]
```diff
@@ -48,6 +48,7 @@
     "mobx-react-lite": "3.2.2",
     "monaco-editor": "0.30.1",
     "react": "17.0.2",
+    "react-icons": "4.3.1",
```
Comment (akphi) [inline packages/legend-extension-external-store-service/package.json:51]: Let's not try to import directly from `react-icons` but use `Icon.tsx` in `@finos/legend-art`

File: packages/legend-extension-external-store-service/src/components/ESService_LegendStudioPlugin.tsx [hunk 2]
```diff
@@ -79,6 +81,28 @@ export class ESService_LegendStudioPlugin
     ];
   }
 
+  getExtraElementIconGetters(): ElementIconGetter[] {
+    return [
+      (type: string): React.ReactNode | undefined => {
+        if (type === SERVICE_STORE_ELEMENT_TYPE) {
+          return (
```
Comment (akphi) [inline packages/legend-extension-external-store-service/src/components/ESService_LegendStudioPlugin.tsx:88]: We genrally don't set color background for our icon so to be consistent let's remove this. Also, we create a generic color for the icon, see `--color-class` for example. :) Honestly, I'm just trying to be consistent here, but I might (likely) regret saying this you know, knowing that we will soon revamp our styling/theming system in the future :sweat:

Comment (hardikmaheshwari) [reply packages/legend-extension-external-store-service/src/components/ESService_LegendStudioPlugin.tsx:88]: @akphi so I requested @gayathrir11 to add this background color to make the icon close to what swagger has so it's easily recognizable. I am totally fine if you believe it's not the best strategy, just wanted to mention this.

Comment (akphi) [reply packages/legend-extension-external-store-service/src/components/ESService_LegendStudioPlugin.tsx:88]: understood but no

File: packages/legend-extension-external-store-service/src/components/ESService_LegendStudioPlugin.tsx [hunk 3]
```diff
@@ -79,6 +81,26 @@ export class ESService_LegendStudioPlugin
...
+            <div
+              className="icon"
+              style={{
+                color: 'var(--color-light-grey-50)',
```
Comment (akphi) [inline packages/legend-extension-external-store-service/src/components/ESService_LegendStudioPlugin.tsx:92]: could you take screenshot of what this look like in the explorer and dropdown menu?

Comment (gayathrir11) [reply packages/legend-extension-external-store-service/src/components/ESService_LegendStudioPlugin.tsx:92]: pic1
pic2

**Review by akphi — APPROVED** [review]

**Pull request merged**